tasks/copy_to_glacier/test/unit_tests/test_handler.py

```diff
@@ -205,6 +201,49 @@ def test_task_empty_granules_list(self):
         s3_cli.head_object.assert_not_called()
         s3_cli.copy.assert_not_called()
 
+    @patch.dict(os.environ, {"ORCA_DEFAULT_BUCKET": ""}, clear=True)
+    def test_task_invalid_environment_variable(self):
+        """
+        Checks to make sure an unset or empty environment variable throws an error
```

Comment:
Strictly speaking, this only checks that an empty environment variable throws an error. If the variable is unset, then the error message will be slightly messier. If you want the same error message in both cases, I recommend os.environ.get(key, None)

Reply:
Updated with `os.environ.get`